endpoints/codacy.ts

```diff
@@ -0,0 +1,96 @@
+import {
+  badgenServe,
+  BadgenServeHandlerArgs as Args,
+  BadgenServeHandlers as Handlers,
+  BadgenServeMeta as Meta
+} from '../libs/badgen-serve'
+// import { coverage as cov, coverageColor } from '../libs/utils'
+import got from '../libs/got'
+import { coverage as cov, coverageColor } from '../libs/utils'
+
+export const meta: Meta = {
+  title: 'Codacy',
+  examples: {
+    '/codacy/c/f0875490cea1497a9eca9c25f3f7774e': 'coverage',
+    '/codacy/c/f0875490cea1497a9eca9c25f3f7774e/dev-master': 'branch coverage',
+    '/codacy/g/f0875490cea1497a9eca9c25f3f7774e': 'code quality',
+    '/codacy/g/f0875490cea1497a9eca9c25f3f7774e/dev-master': 'branch code quality'
+  }
+}
+
+export const handlers: Handlers = {
+  '/codacy/:type<c|g>/:projectId/:branch?': handler
```

Comment:
Let's use `coverage` and `grade` for type, it's meaningful and searchable,
and we can merge that two requests to one:

```
const svg = await got(`${uriBase}/${type}/${projectId}${query}`, { json: false })
```

Reply:
Actually woke up with same thoughts =)
Reworked a bit now.